NEWS.md

```
@@ -12,6 +12,10 @@ This PR splits up the exchanged quantity into `F_lh` and `F_sh`.
 This is helpful because some component models store the two fluxes separately,
 and those that use the combined quantity can easily be updated with the sum.
 
+#### Rename bucket-specific options. PR[#1310](https://github.com/CliMA/ClimaCoupler.jl/pull/1310)
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)